.generator/src/generator/templates/example.j2

```diff
@@ -56,6 +56,19 @@ func main() {
 {%- endfor %}
 	apiClient := common.NewAPIClient(configuration)
 	api := datadog.New{{ context.api_instance.name }}Api(apiClient)
+{%- if context.pagination %}
+	resp, r, err := api.{{ context.api_request.operation_id }}WithPagination(ctx{% if parameters %}, {{ parameters }}{% endif %})
```

Comment:
```suggestion
	resp, _, err := api.{{ context.api_request.operation_id }}WithPagination(ctx{% if parameters %}, {{ parameters }}{% endif %})
```

looks like the second returned variable is callback method to cancel the context and not HTTP response

Reply:
Good catch, thanks.